pycloudlib/gce/tests/test_cloud.py

```diff
@@ -0,0 +1,148 @@
+"""Tests related to pycloudlib.gce.cloud module."""
+import mock
+import pytest
+
+from pycloudlib.cloud import ImageType
+from pycloudlib.gce.cloud import GCE
+
+# mock module path
+MPATH = "pycloudlib.gce.cloud."
+
+
+class FakeGCE(GCE):
```

Comment:
+1 on instrumenting this Fake. We don't want unittests reaching out to a credentialed GCP endpoint if we can avoid it as we'll get rate-limited or throttled over time because of CI load.